builtin/logical/transit/path_keys.go

```diff
@@ -206,6 +206,7 @@ func (b *backend) pathPolicyRead(ctx context.Context, req *logical.Request, d *f
 			"type":                   p.Type.String(),
 			"derived":                p.Derived,
 			"deletion_allowed":       p.DeletionAllowed,
+			"min_version":            p.MinVersion,
```

Comment:
I don't really like the name, I think it's confusing next to min_decryption_version and min_encryption_version. Slightly better might be min_available_version, although that mostly has the same problem. Possibly first_available_version?

Reply:
`min_available_version` is better I suppose. `first_available_version` is slightly confusing.